stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Android.kt

```diff
@@ -646,6 +648,47 @@ class Android {
         }
     }
 
+    /**
+     * @see <a href="https://getstream.io/nessy/docs/chat_docs/unread_counts/unread_channel">Channels</a>
```

Comment:
References with "nessy" only work for us. We need a link to already published page.